src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -185,6 +196,58 @@ protected int doWork() {
                     " already exists.  Delete this file and try again, or specify a different output file.");
         }
 
+        // map for aliases mapping a contig to its' aliases
+        final Map<String, Set<String>> contig2aliases = new HashMap<>();
```

Comment:
In general I wouldn't use numbers in variable names like this, in the place of the word "to". Looking at other Maps in picard, such maps are typically named by the value, not the key, such as `aliases`. In cases where both the key and value are mentioned, one convention is to use `by`, e.g. `aliasesByContig`.